files/similar/update.json

```diff
@@ -0,0 +1,8 @@
+{
+  "packageManager": "github",
+  "name": "similar",
+  "repo": "InputSimilar/similar",
+  "files": {
+    "include": ["similar.eva.js"]
```

Comment:
@jimaek Are you saying `similar.eva.js` this document? I know the use of `eval()`, I use because I see 9lime are using `eval()`; but I think you are allowed to `eval()` in. Use `eval()` similar input compression can help reduce 6kb.
If the `eval()` is not allowed, please respond to me.